src/publisher/highlighter.ts

```diff
@@ -0,0 +1,87 @@
+import type Overlay from "./overlay";
+import type { ReactDevtoolsHook } from "./react-devtools-hook";
+import type { Publisher } from "rempl";
+const HIGHLIGHTER_NS = "highlighter";
+
+export default class Highlighter {
+  private overlay: Overlay;
+  private publisher: Publisher;
+  private hook: ReactDevtoolsHook;
+
+  constructor(
+    hook: ReactDevtoolsHook,
+    publisher: Publisher,
+    overlay: Overlay
+  ) {
+    this.hook = hook;
+    this.publisher = publisher;
+    this.overlay = overlay;
+  }
+
+  startInspect() {
+    window.addEventListener('click', this.onClick.bind(this), true);
+    window.addEventListener('mousedown', this.onMouseEvent, true);
+    window.addEventListener('mouseover', this.onMouseEvent, true);
+    window.addEventListener('mouseup', this.onMouseEvent, true);
+    window.addEventListener('pointerdown', this.onPointerDown.bind(this), true);
+    window.addEventListener('pointerover', this.onPointerOver.bind(this), true);
+    window.addEventListener('pointerup', this.onPointerUp, true);
+  }
+
+  stopInspect() {
+    window.removeEventListener('click', this.onClick, true);
```

Comment:
This will not remove a listener since `onClick` was added as `this.onClick.bind(this)`. The same for `onPointerDown` & `onPointerUp`

Reply:
Fixed